### PR TITLE
Fastlane CI integration

### PR DESCRIPTION
**Pull request by chung-lai**
* Trigger build
* Add fastlane support
* hiding sensitive information

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+  before_all do
+    setup_circle_ci
+    app_store_connect_api_key
```
Comment (buracules) [inline fastlane/Fastfile:21]: How does this works? Did you add environment variables to Circle CI?

**Comment by buracules** [comment]
Fastlane stuff looks good to me. But i am not that experienced with custom scripts.

**Comment by buracules** [comment]
Also i believe we are missing error handling right now if somethings happens we will never now

File: fastlane/Fastfile (hunk 1 above)
Comment (chung-lai) [reply fastlane/Fastfile:21]: yes, they are stored as environment variables in CircleCI and it's one of the fastlane action. Required for CircleCI and fastlane integration

**Review by buracules — APPROVED** [review]

**Comment by chung-lai** [comment]
No code change on this one, it's all install script related. Not sure why it fails the test :)

Merging this, @buracules, please feel free to add any feedback on the error handling.

**Comment by chung-lai** [comment]
This is more like a beta experience, the default option is to build locally. @darinwf, feel free to leave any feedback and we can always make any follow up change.

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+     )
+     slack(
+       message: ":fruitcompany: Uploaded Testflight Build #*#{build_number}* (*v#{marketing_version}*) to <#{ENV["TESTFLIGHT_URL"]}|App Store>",
+       default_payloads: ["git_branch", "last_git_commit", "last_git_commit_hash"],
+       slack_url: ENV["SLACK_URL"]
+     )
+  end
+
+  lane :send_slack_message do |options|
+      v = lane_context[SharedValues::VERSION_NUMBER]
```
Comment (buracules) [inline fastlane/Fastfile:65]: I don't think we need these properties anymore

File: fastlane/Fastfile [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+       clean: true,
+       export_options: "./Scripts/ExportOptions.plist"
+     )
+     upload_to_testflight(
+       changelog: "Internal Test Build",
+       groups: [ENV["TESTFLIGHT_GROUP_1"],
+                 ENV["TESTFLIGHT_GROUP_2"],
+                 ENV["TESTFLIGHT_GROUP_3"]]
+     )
+     slack(
```
Comment (buracules) [inline fastlane/Fastfile:57]: We can also use `send_slack_message` lane here as well

**Pull request merged**